docs/docs/markers.mdx

```diff
@@ -0,0 +1,291 @@
+
+### Conditions
+
+We support the following conditions:
+
+- `action`: the specified bot action was executed.
+- `intent`: the specified user intent was detected.
+- `slot_was_set`: the specified slot was set.
+
+We also support their negated forms:
+
+- `not_action`: the event is not an `ActionExecuted` with the specified action.
+- `not_intent`: the event is not a `UserUttered` with the specified intent.
+- `slot_was_not_set`: the specified slot has not been set.
+
+### Operators
+
+To combine conditions, we provide the following operators:
+
+- `and`: all sub-conditions applied.
+- `or`: any of the sub-conditions applied.
+- `not`: the sub-condition did not apply. `not` can have only 1 condition.
+- `seq`: the list of sub-conditions applied in the specified order with any number of events occurring in-between.
+- `at_least_once`: the listed sub-marker definitions occurred at least once. Only the first occurrence will be marked.
+- `never`: the listed sub-marker definitions never occurred.
+
+### Marker Configuration
+
+To define one or more marker, you need to specify a `yaml` configuration file. Here is an example:
+
+```yaml
+marker_name_provided:
+    slot_was_set: name
+marker_mood_expressed:
+    or:
+        - intent: mood_unhappy
+        - intent: mood_great
+marker_cheer_up:
+    seq:
+        - intent: mood_unhappy
+        - action: utter_cheer_up
+marker_bot_challenged:
+    at_least_once:
+        - intent: bot_challenge
+marker_mood_expressed_and_name_provided:
+    and:
+        - slot_was_set: name
+        - or:
+            - intent: mood_unhappy
+            - intent: mood_great
+```
+
+The top level contains unique user-specified names for each marker. This is a dictionary mapping marker names to marker definitions.
+
+Each marker definition can contain an arbitrary combination of `conditions` and `operators` that map to a valid sub-marker definition.
+
+Each `condition` maps a condition tag to a value. For example, in `marker_name_provided`, the tag `slot_was_set` is mapped to the slot `name` which exists in the `domain.yaml` file.
+
+Each `operator` is a dictionary mapping an operator tag (e.g., `and`) to a list of sub markers, which can be conditions or a nested operator-conditions definition (e.g., see `marker_mood_expressed_and_name_provided`).
+
+:::note
+You cannot yet reuse an existing marker name in the definition of another marker.
+
+:::
+
+## Running the Markers script
+
+Markers are extracted from the dialogues already stored in your [tracker store](https://rasa.com/docs/rasa/next/tracker-stores). Once you've created your Marker definitions in the Makers configuration file, you can extract them by running the follow command:
+
+```bash
+rasa evaluate markers markers.yaml extracted_markers.csv all
```

Comment:
Are we sure this command is correct? 

```
> rasa evaluate markers markers.yaml extracted_markers.csv all

usage: rasa evaluate markers [-h] [-v] [-vv] [--quiet] [--config CONFIG]
                             [--no-stats | --stats-file-prefix [STATS_FILE_PREFIX]]
                             [--endpoints ENDPOINTS] [-d DOMAIN]
                             output_filename {first_n,sample,all} ...
rasa evaluate markers: error: argument strategy: invalid choice: 'extracted_markers.csv' (choose from 'first_n', 'sample', 'all')
```

Also per `--help`. 

```
rasa evaluate markers --help                                

usage: rasa evaluate markers [-h] [-v] [-vv] [--quiet] [--config CONFIG]
                             [--no-stats | --stats-file-prefix [STATS_FILE_PREFIX]]
                             [--endpoints ENDPOINTS] [-d DOMAIN]
                             output_filename {first_n,sample,all}
```

The main positional argument here should be `extracted_markers.csv`, no? That means we shouldn't add `markers.yaml` in this line.

Reply:
```rasa evaluate markers --config markers.yaml extracted_markers.csv all``` should work (for argparse reasons, and because the strategies are subparsers, optional args on the base command have to be before the strategy)